golos.publication/golos.publication.cpp

```diff
@@ -1013,16 +1044,14 @@ void publication::erase_reblog(name rebloger, structures::mssgid message_id) {
 int64_t publication::pay_delegators(int64_t claim, name voter,
         eosio::symbol tokensymbol, std::vector<structures::delegate_voter> delegate_list, structures::mssgid message_id) {
     int64_t dlg_payout_sum = 0;
+    std::vector<eosio::token::recipient> delegators_payout;
```

Comment:
It's better to use one `payto` call for curators & delegators. This reduce inline transaction call count and simplify logic for split bulk's by size (in feature we need do to enable payouts with very large amount of curators).